source/read/project.txt

```diff
@@ -0,0 +1,160 @@
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+Projection Types
+----------------
+
+You can use a projection to specify which fields to include in a return
+document, or to specify which fields to exclude. You cannot combine inclusion and
+exclusion statements in a single projection, unless you are excluding the
+``_id`` field.
+
+Specify Fields to Include
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To specify the fields to include in the result, specify the ``projection``
+option in a parameter to the ``find`` method. To set this option, use the following syntax:
+
+.. code-block:: ruby
+
+   { projection: { <field_name>: 1 } }
+
+The following example uses the ``find`` method to find all restaurants in which the ``name``
+field value is ``'Emerald Pub'``. Then, the code specifies the ``projection`` option
+to instruct the find operation to return only the ``name``, ``cuisine``, and ``borough`` fields
+of matching documents:
+
+.. io-code-block::
+   :copyable:   
+
+   .. input:: /includes/read/project.rb
+      :start-after: start-project-include
+      :end-before: end-project-include
+      :language: ruby
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      {"_id"=>BSON::ObjectId('...'), "borough"=>"Manhattan", "cuisine"=>"American", "name"=>"Emerald Pub"}
+      {"_id"=>BSON::ObjectId('...'), "borough"=>"Queens", "cuisine"=>"American", "name"=>"Emerald Pub"}
+
+When you use a projection to specify fields to include in the return
+document, the ``_id`` field is also included by default. All other fields are
+implicitly excluded. To remove the ``_id`` field from the return
+document, you must :ref:`explicitly exclude it <ruby-project-remove-id>`.
+
+.. _ruby-project-remove-id:
+
+Exclude the ``_id`` Field
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When specifying fields to include, you can also exclude the ``_id`` field from
+the returned document. 
+
+The following example performs the same query as the preceding example but
+excludes the ``_id`` field from the projection:
+
+.. io-code-block::
+   :copyable:   
+
+   .. input:: /includes/read/project.rb
+      :start-after: start-project-include-without-id
+      :end-before: end-project-include-without-id
+      :language: ruby
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      {"borough"=>"Manhattan", "cuisine"=>"American", "name"=>"Emerald Pub"}
+      {"borough"=>"Queens", "cuisine"=>"American", "name"=>"Emerald Pub"}
+
+Specify Fields to Exclude
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To specify the fields to exclude from the result, specify the ``projection``
+option in a parameter to the ``find`` method. To set this option, use the
+following syntax:
+
+.. code-block:: ruby
+
+   { projection: { <field_name>: 0 } }
+
+The following example uses the ``find`` method to find all restaurants in which the ``name``
+field value is ``'Emerald Pub'``. Then, the code uses the ``projection`` option
+to instruct the find operation to omit the ``name`` and ``address`` fields
```

Comment:
[i] The code example uses `grades` here, so can update description or code example to match.

Reply:
oops thanks for catching!